mail/dovecot2/Portfile

```diff
@@ -43,9 +43,18 @@ patch.pre_args      -p1
 patchfiles          patch-doc-example-config-conf.d-10-master.conf.diff \
                     patch-src-master-master-settings.c.diff
 
-checksums           rmd160  37230c065284899fa107ab1c91831ca154f230ac \
-                    sha256  3b61c128f3ef947f5d3e8b1113c321d0597600a3102a8e5b405484a7c049eb51 \
-                    size    3653354
+checksums           patch-doc-example-config-conf.d-10-master.conf.diff \
+                    rmd160  d8968a93871950264e7e43de6da8635d69676137 \
+                    sha256  8dbf8d13224df2f562f6010bad0f6c947d2cbf8988ff74e0134159cfc465e1d2 \
+                    size    3653398 \
+                    patch-src-master-master-settings.c.diff \
+                    rmd160  d8968a93871950264e7e43de6da8635d69676137 \
+                    sha256  8dbf8d13224df2f562f6010bad0f6c947d2cbf8988ff74e0134159cfc465e1d2 \
+                    size    3653398 \
+                    core-2.3.7.2.tar.gz \
```

Comment:
Use ${version} here instead of hardcoding.

Reply:
Yah, good idea.

Reply:
```suggestion
                    core-${version}.tar.gz \
```